_ml-commons-plugin/api/model-group-apis/get-model-group.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: Get model group
+parent: Model group APIs
+grand_parent: ML Commons APIs
+nav_order: 30
+---
+
+# Get a model group
+
+This API is available from 2.13 release
+{: .important}
+
+If model access control is enabled on your cluster, only the owner or users with matching backend roles can get the model group. Any users can get any public model group.
+
+If model access control is disabled on your cluster, users with the `get model group API` permission can get any model group.
+
+Admin users can get any model group.
```

Comment:
Does hidden model has model group ?

Reply:
Yes but model group is not hidden. We don't have hidden model group.

Reply:
Got it, so admin can still get hidden model group

Reply:
yes